### ISSUE TITLE
[Feature Request] Win10-style Start Menu?

### ISSUE THREAD

**Issue by Darthagnon** [Open-Shell/Open-Shell-Menu#412]
**Describe the solution you'd like**
Would it be possible to implement a Windows 10-style start menu in Open Shell? Ideally it would look like the current Windows 10 start menu, but function on older versions of Windows, though potentially skinning like this would be possible:

![Win10 Start Menu on Win7 concept](https://twitter.com/Futur3Sn0w/status/1290727606942408705)

**Comment by Ibuprophen** [comment]
@Darthagnon, After a quick Google search, I'm hoping that one or more of the following 3 links has what you're looking for (or close enough for you to customize the rest from). 

https://github.com/davestewart/classic-shell-win10

http://www.classicshell.net/forum/viewtopic.php?f=17&t=7268

https://winaero.com/blog/best-skins-for-classic-shells-start-menu/

Good Luck! 


~Ibuprophen

**Comment by Darthagnon** [comment]
Thank you so much for those links; the second one looks closest to what I had in mind, and very promising. Will test it!

**Comment by bonzibudd** [comment]
Because of Open-Shell's limitations, there is no _easy_ end-user way of pinning and unpinning for what you are talking about. The "Pin to Start Menu (Open-Shell)" option will always just pin items to the left-hand side.

**Comment by Darthagnon** [comment]
Thank you for the confirmation! That's a pity. Maybe one day...

**Comment by Ibuprophen** [comment]
I had appropriately tagged this issue and, if there's no objections, I'll keep it open for the time being in case someone else inquires about this feature. 

If so, that issue would be referred to this one and closed as a previously requested feature. 


~Ibuprophen

**Comment by Ibuprophen** [comment]
I had appropriately tagged this issue and, if there's no objections, I'll keep it open for the time being in case someone else inquires about this feature. 

If so, that issue would be referred to this one and closed as a previously requested feature. 


~Ibuprophen

**Comment by Ibuprophen** [comment]
I had appropriately tagged this issue and, if there's no objections, I'll keep it open for the time being in case someone else inquires about this feature. 

If so, that issue would be referred to this one and closed as a previously requested feature. 


~Ibuprophen

**Comment by Darthagnon** [comment]
[Someone on Reddit](https://old.reddit.com/r/Windows10/comments/iaxft2/if_the_windows_7_start_menu_survived_in_windows_10/) made a [skin](https://1drv.ms/u/s!AqxEMH2XuF6DxGRlhpsY6KPWKrSz?e=Uobrzc) that is a very nice Start Menu for Win10 (though it still doesn't implement tiles, it looks very Fluent), though still not 100% what I'm looking for. Sharing here for future readers:

![picture](https://i.redd.it/a8y2jocopeh51.png)

**Comment by CTVCAM8** [comment]
What goes around comes around. The round corners are back in style like tabs in the new Edge.

That ginger_bread84 guy just needs a little of his same round treatment on the Search_background_mask
along with changes in this picture: 

![SearchR](https://user-images.githubusercontent.com/44127216/90440741-99499100-e08c-11ea-89ae-a04867890ba6.png)


Would not work for small icons option though as there is only 1 pixel protected corners.

**Comment by bonzibudd** [comment]
@CTVCAM8 

That skins actually mine haha. It was made more as a concept but ended up putting a link because many people were interested. If I do a follow-up I will certainly implement a better search mask.

**Comment by Ibuprophen** [comment]
Please correct me if I'm misunderstanding this open issue... 

When it comes the look, skin, etc... of the Start Menu, i don't believe that this would constitute a "Feature Request" if it falls into a category of the customization & appearance of the start menu. 

I don't intend to discourage someone asking about an available skin and such to alter the appearance. I do encourage the community asking for assistance to/from other members. 

I would like to clarify that the start menu already has the capability to alter its appearance (hence the feature is available). 

It's just that others need to research/look into the understanding of what to do/not to do when it comes to altering those certain settings and such to accomplish this look. 

I've always felt that a "Feature Request" really fits within the realm of having, creating, fixing, etc... something that not available or is available, but doesn't work that requires one or more Classic Shell team members to look into it. 

I'm not trying to rant nor complain, but to clarify this and, if I'm wrong, please correct me where necessary. 

I truly hope that I had explained this okay via text as I only posted this with good intentions. 


~Ibuprophen

**Comment by CTVCAM8** [comment]
Ah-ha! The trap was set, the bait got bitten.

Yeah, the compact view search box has some wiggle room horizontally, but only 1 pixel vertically so it will stretch the corner.
You could have a separate Search_background_mask for that but just have the one corner pixel 0 or 64/255 alpha 


![Small icons search box](https://user-images.githubusercontent.com/44127216/90447712-b84e2000-e098-11ea-98b1-6c0b4949f4a2.png)

**Comment by CTVCAM8** [comment]
Should we be using the Discussion room for something like this?

**Comment by bonzibudd** [comment]
I posted a follow-up for the rounded skin on the UC-Menu-Skins room.

**Comment by Darthagnon** [comment]
Guess it's kinda off-topic, but @bonzibudd Double-checking, is [this](https://github.com/bonzibudd/Fluent-Metro) the latest version of your modern Classic Shell skin? Do you know if it works on Windows 7 (readme doesn't mention)?

It's the closest we've currently got to a Windows 10-style skin for OpenShell

**Comment by bonzibudd** [comment]
@Darthagnon 

Yes, that's the latest release so far. Currently it doesn't work on 7 due to high reliance on metro colors, but I am planning on making it work eventually. I'll consider that for an upcoming preview version.

**Comment by Darthagnon** [comment]
Thank you so much! Is it worth me opening a GH Issue about it, or mentioning Win7 in https://github.com/bonzibudd/Fluent-Metro/issues/6 ?

**Comment by bonzibudd** [comment]
I don't have an issue template yet, but you can make a new issue if it will let you. The issue you mentioned is separate, but if you can't make a new one you can mention it there.

**Comment by bonzibudd** [comment]
@Darthagnon 

Just thought I would reference this here, since it is very similar to the original request. I've been working on a version of my skin in the two-column variant, which has support for customizable tiles:
![image](https://user-images.githubusercontent.com/61938331/98316345-3dfeaf00-1fa8-11eb-8c5a-7234f2578b04.png)

It will be on the same page as my current skin within a zip file, along with an xml file for easy configuration. I'm not sure if you're still interested in tiles specifically, but I think it is fitting here.

**Comment by Darthagnon** [comment]
@bonzibudd That's the sort of thing I was talking about and hoping for! Tiles and all! Thank you so much! It's beautiful!

**Comment by bonzibudd** [comment]
@Darthagnon 

I've posted a newer version, should be more stable and have some more options. I probably won't post anything else on this thread. 

If that XML file doesn't work, you may need to select the skin manually.
